app/screens/authentication-screen/biometry-check-screen.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react"
+import { Image } from "react-native"
+import EStyleSheet from 'react-native-extended-stylesheet'
+
+// Components
+import { Screen } from "../../components/screen"
+
+// Constants
+import { palette } from "../../theme/palette"
+
+// Utils
+import { isSensorAvailable } from "../../utils/biometricAuthentication"
+
+// Types
+import type { ScreenType } from '../../types/screen'
+
+const BitcoinBeachLogo = require("../get-started-screen/bitcoinBeach3.png")
+
+const styles = EStyleSheet.create({
+  container: {
+    alignItems: "center",
+    flex: 1,
+    width: "100%",
+  },
+
+  Logo: {
+    maxHeight: "50%",
+    maxWidth: "50%",
+    marginTop: 24,
+  }
+})
+
+type Props = {
+  navigation: any,
+}
+
+export const BiometryCheckScreen: ScreenType = ({ navigation }: Props) => {
+
+  React.useEffect(() => {
+    checkIfBiometryIsAvailable()
+  })
```

Comment:
this can probbaly be written as `React.useEffect(() => checkIfBiometryIsAvailable)`

Reply:
Is there a reason you prefer it setup like this? The rest of the app seems to use React.useEffect(() => {}).